apps/passport-client/components/screens/ImportBackupScreen.tsx

```diff
@@ -0,0 +1,386 @@
+import { EmailPCDTypeName } from "@pcd/email-pcd";
+import { deserializeStorage } from "@pcd/passport-interface";
+import { Spacer, styled } from "@pcd/passport-ui";
+import { PCDCollection } from "@pcd/pcd-collection";
+import { PCD } from "@pcd/pcd-types";
+import { SemaphoreGroupPCDTypeName } from "@pcd/semaphore-group-pcd";
+import { SemaphoreIdentityPCDTypeName } from "@pcd/semaphore-identity-pcd";
+import { useCallback, useEffect, useState } from "react";
+import { useFilePicker } from "use-file-picker";
+import { useDispatch, usePCDCollection } from "../../src/appHooks";
+import { getPackages } from "../../src/pcdPackages";
+import { AppState } from "../../src/state";
+import { useSelector } from "../../src/subscribe";
+import { Button, H2 } from "../core";
+import { MaybeModal } from "../modals/Modal";
+import { AppContainer } from "../shared/AppContainer";
+import { ScreenNavigation } from "../shared/ScreenNavigation";
+
+export function useImportScreenData() {
+  return useSelector<AppState["importScreen"]>((s) => s.importScreen, []);
+}
+
+const NoFolderSymbol = Symbol("None");
```

Comment:
Maybe "Root" or "Main Folder" would communicate this a bit clearer to an end-user